docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc

```diff
@@ -0,0 +1,23 @@
+= How to Manually Perform a Redis Failover
+
+In certain situations Redis nodes end up in `CrashLoopBackOff` because they receive a wrong (old) leader IP address from the running Sentinel.
+
+In the logs this might look like:
+
+[source,shell]
+----
+sentinel Could not connect to Redis at 10.0.0.20:26379: Connection refused
+----
+
+To resolve this situation, get the IP address of the running leader, connect to a Redis Sentinel Pod, start a `redis-cli` and initiate the failover.
+
+[source,shell]
+----
+kubectl get pod -o wide
+
+kubectl exec redis-node-0 sentinel -it -- sh
+redis-cli -a $(< $REDIS_PASSWORD_FILE) -p 26379
```

Comment:
Does also not work:

```
mh $ kubectl exec redis-node-0 -c sentinel -it -- sh
$ redis-cli -a $(< $REDIS_PASSWORD_FILE) -p 26379
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
Warning: AUTH failed
(error) ERR unknown command `26379`, with args beginning with:
$
$
```

```suggestion
export REDISCLI_AUTH="$(< $REDIS_PASSWORD_FILE)"
redis-cli -p 26379
```

Reply:
Oh, `< file` only works in `bash`, so either replace `<` with `cat` or `sh` (above) with `bash`